src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final Color[] arrayColors = Color.values();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(arrayColors.length - 1);

```

Comment:
Please, reach out [documentation](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#nextInt-int-) for nextInt(int bound) method. For now, you are excluding the last color.

Reply:
The problem here is actual